vmdb/spec/views/host/_form.html.haml_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "spec_helper"
+include ApplicationHelper
```

Comment:
OK, we'll have to tackle that issue later, including ApplicationHelper globally will burn us badly someday.